<launch>
  <!-- Publish a static transform from /map to /odom -->
  <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_map_to_odom" args="0 0 0 0 0 0 /map /odom" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_odom_to_cam" args="0 0 0 0 0 0 /odom /camera_init" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_body_to_livox" args="0 0 0 0 0 0 /body /livox_frame" />

  <!-- Include the PS3 joystick launch file -->
  <include file="$(find ps3joy)/launch/ps3.launch" />

  <node pkg="base_controller" type="pure_manual" name="pure_manual" output="screen">
    <param name="lookahead_distance" value="1.5"/>
    <param name="min_lookahead_distance" value="0.2"/>
    <param name="wheel_base" value="0.36"/>
    <param name="odometry_topic" value="/odom"/>
    <param name="path_topic" value="/loaded_path"/>

    <param name="max_speed" value="1.5"/>
    <param name="min_speed" value="0.3"/>
    <param name="max_steering_angle" value="0.5"/>
  </node>
</launch>
